rules/CodeQuality/Rector/Class_/CompleteDynamicPropertiesRector.php

```diff
@@ -154,7 +154,9 @@ private function shouldSkipClass(Class_ $class): bool
             return true;
         }
 
-        return $class->extends instanceof FullyQualified && ! $this->reflectionProvider->hasClass($class->extends->toString());
+        return $class->extends instanceof FullyQualified && ! $this->reflectionProvider->hasClass(
```

Comment:
When I ran `composer complete-check` it found several unrelated code standard fixes. I have included them in the PR so that all tests pass even if they are unrelated to the current code change